clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/TitusServerGroupNameResolver.groovy

```diff
@@ -41,8 +41,14 @@ class TitusServerGroupNameResolver extends AbstractServerGroupNameResolver {
   @Override
   List<AbstractServerGroupNameResolver.TakenSlot> getTakenSlots(String clusterName) {
     def clusterNameParts = Names.parseName(clusterName)
+
+    // The sequence does not exist as part of the clusterName.  Therefore, if we have a sequence,
+    // this is because frigga parsed the detail as a sequence.
+    def cluster = clusterNameParts.sequence ?
+      clusterNameParts.cluster + "-v${clusterNameParts.sequence}" : clusterNameParts.cluster
+
     List<Job> jobs = titusClient.findJobsByApplication(clusterNameParts.app)
-      .findAll { Names.parseName(it.name).cluster == Names.parseName(clusterName).cluster }
+      .findAll { Names.parseName(it.name).cluster == cluster }
```

Comment:
Suspect this could just be Names.parseName(it.name) == clusterName and get rid of the logic above to derive cluster